scripts/prepare_cordana.sh

```
@@ -0,0 +1,25 @@
+# echo -------------------------
+# ls -al ~
+# echo -------------------------
+# chmod +x ~/qt-installer.run
+# echo -------------------------
+# ls -al ~
+# ~/qt-installer.run --root ~/qt_install_dir --email $ENV_QT_EMAIL --pw $ENV_QT_PASSWORD --accept-licenses --default-answer --confirm-command --accept-obligations install qt.qt6.680.linux_gcc_64
+# ls ~/qt_install_dir/6.8.0
+# echo -------------------------
+# ls ~/qt_install_dir/6.8.0/gcc_64/bin
+mkdir -p build
+cd build
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)